packages/runtime/container-runtime/package.json

```diff
@@ -158,6 +158,9 @@
       "TypeAliasDeclaration_SummarizerStopReason": {
         "forwardCompat": false,
         "backCompat": false
+      },
+      "ClassDeclaration_ScheduleManager": {
```

Comment:
Can tests using ScheduleManager be moved into container runtime and thus remove a need to export ScheduleManager out of a package?

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/1981